packages/web-sdk/src/index.ts

```diff
@@ -14,6 +14,7 @@ export {
   parseStacktrace,
   ViewInstrumentation,
   WebVitalsInstrumentation,
+  WebVitalsWithAttributionInstrumentation,
```

Comment:
Another point is that I didn't find where I can update the SDK documentation, if you can help me with this point